### PR TITLE
Add links to User Support Documentation

### LINKED ISSUE #16132: Surface user handbook in editor interface
Previously: #11252

Relevant Slack conversations ([links require registration](https://make.wordpress.org/chat/)):

- https://wordpress.slack.com/archives/C02QB2JS7/p1558531784196900
- https://wordpress.slack.com/archives/C02RQBWTW/p1560372677442600

A block editor user handbook can be reached at the following URL:

https://wordpress.org/support/article/wordpress-editor/

However, there are not very obvious paths for how a user might reach this documentation.

We should consider how to present these guides to a user who is becoming acclimated with the block editor.

Options to consider, amongst other possibilities:

- As part of the "New User" tips which appear upon first load of the editor, include a link to the user documentation
- Include a "Help" menu link under the "More tools & options" menu in the top-right, which directs to this navigation
- Reincorporate the "Help" panel which appears at the top of other administration screens ([screenshot](https://user-images.githubusercontent.com/1779930/59386300-4e1f4a00-8d34-11e9-85ac-e2742ca5d738.png))

### PR DESCRIPTION
**Pull request by mkaz**
## Description

There is excellent user documentation available at: https://wordpress.org/support/article/wordpress-editor/
This PR adds links within the Block Editor and documentation to the user docs.
Fixes #16132

- Add link on first page of Block Editor Handbook
- Add new page to Welcome Guide with link (opens in new window)
- Add new "Help" link in Tools dropdown menu (opens in new window)

**Design needed for Welcome guide**

Screenshot from 2019-12-29 10-48-31

## How has this been tested?

Confirm links work in new places, display properly and makes sense in context

## Types of changes

Add links to documentation.

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR. .

### CODE REVIEW

**Comment by mkaz** [comment]
@melchoyce This relates to your issue https://github.com/WordPress/gutenberg/issues/18952

I added one link to the top-level page in the handbook, but welcome any additional spots or ideas for adding more links to user docs.

File: packages/edit-post/src/components/welcome-guide/index.js [hunk 1]
```diff
@@ -67,6 +67,18 @@ export default function WelcomeGuide() {
 				</p>
 			</GuidePage>
 
+			<GuidePage className="edit-post-welcome-guide__page">
+				<h1 className="edit-post-welcome-guide__heading">
+					{ __( 'Detailed Documentation' ) }
+				</h1>
+				<p className="edit-post-welcome-guide__text">
+					{ __( 'For detailed information on using the block editor, see the ' ) }
+					<a target="_new" href="https://wordpress.org/support/article/wordpress-editor/">
```
Comment (Soean) [inline packages/edit-post/src/components/welcome-guide/index.js]: Links should also be translatable, then we can link the localized version if there is one.

Comment (mkaz) [reply packages/edit-post/src/components/welcome-guide/index.js]: :+1: Fixed, also switched it to use ExternalLink component

**Comment by benhuberman** [comment]
Hey @mkaz -- is the copy that needs a review the language included in the [screenshot](https://user-images.githubusercontent.com/45363/71561291-65502b80-2a29-11ea-95ab-9ad65ad7d4a2.png) you've added?

If so, here's another stab at that:

> **Learn how to use the block editor**
New to the block editor? Want to learn more about using it? [link]Here's a detailed guide.[/link] 

Happy to tweak further (or to look at other copy if that's not what you were referring to).

**Comment by mkaz** [comment]
@benhuberman yes, that is the copy I was looking for. :+1:
What you provided works for me, thanks so much!

**Comment by mapk** [comment]
@mkaz, I made a few design tweaks to help this along. I think it's ready to go now.

Screen Shot 2019-12-31 at 10 38 07 AM

**Review by mapk — APPROVED** [review]
Looks good! 👍 Let's get it merged.

**Pull request merged**